SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/EntityComponentOpList.h

```diff
@@ -45,7 +47,7 @@ class EntityComponentOpListBuilder
 																	 const FString& Message);
 
 	const char* StoreString(FString Message) const;
-	const Worker_Entity* StoreQueryEntities(TArray<Worker_Entity> Entities);
+	const Worker_Entity* StoreQueriedEntities(TArray<Worker_Entity> Entities) const;
```

Comment:
This should be taking a TArray<TArray<ComponentData>> or similar. Right now this make ownership unclear. Presumably the data pointed to by each Worker_Entity isn't owned by the op list.